From 35ea816d3c9ca201cb5ab301bcca01446bdf4f5b Mon Sep 17 00:00:00 2001
From: Thomas Ieong <th.ieong@free.fr>
Date: Fri, 30 May 2025 15:40:10 +0200
Subject: [PATCH] Include lxcfs, lxc common.conf and common.seccomp

---
 src/PVE/LXC.pm | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/PVE/LXC.pm b/src/PVE/LXC.pm
index c333230..8d8c855 100644
--- a/src/PVE/LXC.pm
+++ b/src/PVE/LXC.pm
@@ -672,6 +672,13 @@ sub update_lxc_config {
     my $inc = "$cfgpath/$ostype.common.conf";
     $inc ="$cfgpath/common.conf.d/01-pve.conf" if !-f $inc;
     $raw .= "lxc.include = $inc\n";
+    # Include lxcfs, need to have virtualisation.lxcfs enabled
+    $raw .= "lxc.include = @lxcfspath@/share/lxc/config/common.conf.d/00-lxcfs.conf\n";
+    # Include common.conf
+    $raw .= "lxc.include = @lxcpath@/share/lxc/config/common.conf\n";
+    # Include common.seccomp
+    $raw .= "lxc.include = @lxcpath@/share/lxc/config/common.seccomp\n";
+
     if ($unprivileged) {
 	$inc = "$cfgpath/$ostype.userns.conf";
 	$inc = "$cfgpath/userns.conf" if !-f $inc;
-- 
2.49.0

