From d12b6fe9d3e942679362b69cfff4b37a3da924bc Mon Sep 17 00:00:00 2001
From: Thomas Ieong <th.ieong@free.fr>
Date: Thu, 29 May 2025 12:33:58 +0200
Subject: [PATCH] fix: force detach lxc rootdir

---
 src/PVE/LXC.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/PVE/LXC.pm b/src/PVE/LXC.pm
index 2b9f0cf..119be2a 100644
--- a/PVE/LXC.pm
+++ b/PVE/LXC.pm
@@ -1561,7 +1561,7 @@ sub umount_all {
 	return if !PVE::ProcFSTools::is_mounted($mount_path);
 
 	eval {
-	    PVE::Tools::run_command(['umount', '-d', $mount_path]);
+	    PVE::Tools::run_command(['umount', '-d', '-v', '-f', '-l', $mount_path]);
 	};
 	if (my $err = $@) {
 	    if ($noerr) {
-- 
2.49.0

