From 7df813a3143eabcdfa8a1f01eeb32b26cde4f138 Mon Sep 17 00:00:00 2001
From: Thomas Ieong <th.ieong@free.fr>
Date: Fri, 30 May 2025 18:11:36 +0200
Subject: [PATCH] net

---
 src/PVE/Network.pm | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/PVE/Network.pm b/src/PVE/Network.pm
index 317e1a9..36e3fbb 100644
--- a/PVE/Network.pm
+++ b/PVE/Network.pm
@@ -210,7 +210,9 @@ sub iface_set :prototype($@) {
 sub iface_set_master :prototype($$) {
     my ($iface, $master) = @_;
     if (defined($master)) {
-	eval { iface_set($iface, 'master', $master) };
+	#eval { iface_set($iface, 'master', $master) };
+	run_command(['/sbin/ip', 'link', 'set', $iface, 'master', $master]);
+	print "IFACE IS TESTING $iface" . "$master" .   "\n";
 	die "can't enslave '$iface' to '$master'\n" if $@;
     } else {
 	eval { iface_set($iface, 'nomaster') };
-- 
2.49.0

